ios-dep-8-0-arm64-armv7-hid-sections-lto-cxx11.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2017-2018, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_DEP_8_0_ARM64_ARMV7_HID_SECTIONS_LTO_CXX11_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_DEP_8_0_ARM64_ARMV7_HID_SECTIONS_LTO_CXX11_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_module_path.cmake")
+include(polly_clear_environment_variables)
+include(polly_init)
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone-default-sdk.cmake") # -> IOS_SDK_VERSION
+
+set(IOS_DEPLOYMENT_SDK_VERSION 8.0)
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (arm64 armv7) / \
+${POLLY_XCODE_COMPILER} / \
+c++11 support / hidden / data-sections / function-sections / LTO"
+    "Xcode"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
```

Comment:
Not needed, it's included by line below.

Reply:
Changed in the new commits.